client/src/components/content/PropertiesPanel/PropertiesPanelBuilder/PropertiesPanelBuilder.jsx

```diff
@@ -87,27 +95,30 @@ export default class PropertiesPanelBuilder extends Component {
    * @description Fetch all applications from MongoDB and save in session storage.
    */
   async saveAvailableApplicationsToSessionStorage() {
-    await fetch('/get-available-applications')
+    await fetch('/rpa-framework/commands/get-available-applications')
       .then((response) => response.json())
-      .then(data => {
+      .then((data) => {
         sessionStorage.setItem('AvailableApplications', data);
-      })
+      });
   }
 
   /**
-   * @description 
+   * @description
    * Checks if tasks for selected application are already stored in session storage.
    * Otherwise, fetch tasklist from MongoDB.
    * @param {*} selectedApplication Application for which to get the tasks for.
    */
   async getTasksForApplication(selectedApplication) {
-    let currentSavedTasksObject = JSON.parse(sessionStorage.getItem('TaskToApplicationCache'));
+    let currentSavedTasksObject = JSON.parse(
+      sessionStorage.getItem('TaskToApplicationCache')
+    );
 
     if (selectedApplication in currentSavedTasksObject) {
       this.setState({
         selectedApplication: selectedApplication,
-        tasksForSelectedApplication: currentSavedTasksObject[selectedApplication],
-        disableTaskSelection: false
+        tasksForSelectedApplication:
+          currentSavedTasksObject[selectedApplication],
```

Comment:
I don't see how this linebreak increases readability

Reply:
This is done by my vs code formatting on save. I would argue it is not a problem.